recipe/meta.yaml

```diff
@@ -0,0 +1,181 @@
+{% set name = 'hail' %}
+{% set version = '0.2.127' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # the build process requires a repository to set some variables
+  # (e.g. version)
+  git_url: https://github.com/hail-is/hail.git
+  git_rev: {{ version }}
+  patches:
+    - patches/0001-remove-vendored-libsimdpp.patch
+    - patches/0002-remove-vendored-catch2.patch
+    - patches/0003-patch-requirements.patch
+    - patches/0004-pluggy-old-style-wrappers.patch
+    - patches/0005-patch-spark-3.3.2-to-3.4.1.patch
+
+build:
+  number: 0
+  # avro and others a not available for win
+  # openjdk and pyspark not available for s390x
+  skip: true  # [py<39 or win or py>311 or s390x]
+  skip: true  # [blas_impl == "mkl"]
+  entry_points:
+    - hailctl = hailtop.hailctl.__main__:main
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make      # [linux]
+    - patch     # [not win]
+    - m2-patch  # [win]
+    - git
+  host:
+    - python
+    - pip
+    - setuptools
+    # prerequisites
+    - pytest-runner
+    - pyspark >=3.3.2,<=3.4.1
+    - openjdk 11.*
+    - lz4
+    # blas (lapack is brought in along)
+    - openblas-devel {{ openblas }}  # [blas_impl == "openblas"]
+    # libsimdpp and catch2 unvendored dependencies
+    - libsimdpp
+    - catch2
+    # hail build needs these here to build hailtop/hailctl
+    - aiodns >=2.0.0,<3
+    - aiohttp >=3.9,<4
+    - azure-identity >=1.6.0,<2
+    - azure-mgmt-storage ==20.1.0
+    - azure-storage-blob >=12.11.0,<13
+    - boto3 >=1.17,<2.0
+    - botocore >=1.20,<2.0
+    - dill >=0.3.6,<0.4
+    - frozenlist >=1.3.1,<2
+    - google-auth >=2.14.1,<3
+    - google-auth-oauthlib >=0.5.2,<1
+    - humanize >=1.0.0,<2
+    - janus >=0.6,<1.1
+    - nest-asyncio >=1.5.8,<2
+    - orjson >=3.6.4,<4
+    # - protobuf ==3.20.2
+    # we have protobuf 3.20.3
+    - protobuf >=3.20.2,<=3.20.3
+    # - rich >=12.6.0,<13
+    # we have rich 12.5.0
+    - rich >=12.5.0,<13
+    - typer >=0.9.0,<1
+    - python-json-logger >=2.0.2,<3
+    - pyyaml >=6.0,<7.0
+    - sortedcontainers >=2.4.0,<3
+    - tabulate >=0.8.9,<1
+    - uvloop >=0.19.0,<1  # [not win]
+    - jproperties >=2.1.1,<3
+  run:
+    - python
+    # prerequisites
+    - openjdk 11.*
+    # blas (lapack is brought in along)
+    - openblas-devel {{ openblas }}  # [blas_impl == "openblas"]
```

Comment:
```suggestion
```
openblas-devel has a run export for libopenblas, which is what you actually need at runtime